shared/src/address/mod.rs

```diff
@@ -385,6 +385,19 @@ pub(crate) fn from_leb_bytes(bz: &[u8]) -> Result<u64, Error> {
     Ok(id)
 }
 
+/// Returns an address hash for given data
```

Comment:
Clippy complained about this function being placed after the test module (which, IMO, is a reasonable complaint as tests usually come last).